odl/tomo/operators/ray_trafo.py

```diff
@@ -528,6 +512,75 @@ def adjoint(self):
         return self._adjoint
 
 
+def proj_space_weighting(reco_space, geometry):
+    """Determine a weighting for the range of a ray transform.
+
+    If ``reco_space`` is weighted with the cell sides, and if ``geometry``
+    represents an acquisition with a continuous curve or surface of motion,
+    then the projection space will be weighted per axis, where in
+    the angle axis (axes), non-uniform sampling is allowed and taken
+    into account, and in the detector axis (axes), uniform sampling is
+    mandatory.
+    For a 1D angle partition, the weights are given by ::
+
+        angle_weights[1:-1] = (angles[2:] - angles[:-2]) / 2
```

Comment:
Shouldn't this simply be the cell size for the partition?

Reply:
I guess so, I'll check.

Reply:
No, it's the average between consecutive cells. For non-uniform angles, that makes a difference.